workspaces/global-floating-action-button/.changeset/sweet-clocks-move.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-global-floating-action-button': major
```

Comment:
I would start with a patch release 0.0.1 until we're comfortable. I expect that we will have more then just this PR to have it ready.